docs/pages/guides/localization.mdx

```diff
@@ -0,0 +1,385 @@
+
+Here are some libraries that can be a good option depending on your needs:
+  
+- [`React i18next`](https://react.i18next.com/)
+
+  `React i18next` is a stable, well-maintained library based on `i18next`.
+
+- [`typesafe-i18n`](https://github.com/ivanhofer/typesafe-i18n)
+
+  `typesafe-i18n` compiles translations so that it saves space in the production bundle. It also generates typescript types for your translations, so that you can use them in your code and get autocomplete and type safety.
+  It requires the `Intl` API for some features, so it's only usable in projects with Hermes enabled.
+
+</Collapsible>
+
+### Translating app metadata
+
+If you plan on shipping your app to different countries, or regions, or want it to support various languages, you can provide [localized](/versions/latest/sdk/localization) strings for things like the display name and system dialogs. All of this is easily set up [in your `app.json`](/workflow/configuration) file. First, set `ios.infoPlist.CFBundleAllowMixedLocalizations: true`, then provide a list of file paths to `locales`.
+
+```json app.json
+{
+  "expo": {
+    "ios": {
+      "infoPlist": {
+        "CFBundleAllowMixedLocalizations": true
+      }
+    },
+    "locales": {
+      "ja": "./languages/japanese.json"
+    }
+  }
+}
+```
+
+The keys provided to `locales` should be the [language identifier](https://developer.apple.com/documentation/xcode/choosing-localization-regions-and-scripts), made up of a [2-letter language code](https://www.loc.gov/standards/iso639-2/php/code_list.php) of your desired language, with an optional region code (e.g. `en-US` or `en-GB`), and the value should point to a JSON file that looks something like this:
+
+```json japanese.json
+{
+  "CFBundleDisplayName": "こんにちは",
+  "NSContactsUsageDescription": "日本語のこれらの言葉"
+}
+```
+
+Now, iOS knows to set the display name of your app to `こんにちは` whenever it's installed on a device with the language set to Japanese.
+
+## Enabling RTL Support
+
+Several regions around the world write text from right to left. If you want to localize your app so it looks as expected in RTL languages, you need to make sure your app handles these layout and text direction changes accordingly.
+
+Expo will have full support for RTL applications in SDK48.
+To enable it in your application, install `expo-localization` and add the following properties in your `app.json`
+
+<Terminal cmd={['$ npx expo install expo-localization']} />
+
+```json
+{
+"expo": {
+    "extra": {
+        "supportsRTL": true
+    },
+    "plugins": ["expo-localization"],
+	}
+}
+```
+
+This will enable RTL both when your app is loaded in Expo Go, as well as in Expo dev Client and in applications built using EAS Build or  `expo prebuild`.
```

Comment:
```suggestion
This will enable RTL when your app is loaded in Expo Go, in Expo dev Client, and in applications built using EAS Build or `expo prebuild`.
```